### PR TITLE
03/11/21 refresh

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -55,6 +55,12 @@ kubectl get crd | grep certificates.certmanager
 [#web-console-known-issues]
 == Web console known issues
 
+[#search-result-node]
+=== Node discrepancy between Cluster page and search results
```
Comment (dockerymick) [inline release_notes/known_issues.adoc:59]: Can you italicize Cluster in the title section?

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -55,6 +55,12 @@ kubectl get crd | grep certificates.certmanager
...
+// 2.0:9987
+
+You might see a discrepancy between the nodes dispalyed on the _Cluster_ page and the _Search_ results.
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: Is there a specific discrepancy? Or will the data just not match?

**Review by dockerymick — APPROVED** [review]
Added some comments, but it looks good and there are no conflicts

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (cadawson) [reply release_notes/known_issues.adoc]: Not sure. I am only the refresher on this one.

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (cadawson) [reply release_notes/known_issues.adoc:59]: Not typically in the title, I don't think.

**Pull request merged**